pkg/util/common.go

```diff
@@ -42,3 +43,9 @@ func (t Tree) String() string {
 func IsFilePath(path string) bool {
 	return strings.Contains(path, "/") || strings.Contains(path, ".")
 }
+
+// IsHTTPURL checks whether the given URL is a HTTP URL, empty path or relative URLs would be rejected.
+func IsHTTPURL(path string) bool {
+	u, err := url.Parse(path)
```

Comment:
I'm not sure if this is restrictive enough. I think we'd also want to enforce a http:// or https:// prefix.